api/src/main/java/org/openmrs/api/impl/ObsServiceImpl.java

```diff
@@ -95,9 +95,7 @@ public Obs saveObs(Obs obs, String changeMessage) throws APIException {
 		if(obs.getId() != null && changeMessage == null){
 			throw new APIException("Obs.error.ChangeMessage.required", (Object[]) null);
 		}
-
-		handleExistingObsWithComplexConcept(obs);
```

Comment:
So we move this so instead of happening here, it happens further along in the saveExistingObs and saveNewOrVoidedObs method.  This is so it happens *after* the obs cloning happens in the saveExistingObs method.

There is one other branch the code can take--"saveObsNotDirty", but this seems to only happen when adding members to an existing grouping obs, so I don't think we need to handle complex obs in this case.